### DISCUSSION TITLE
Question : Does it possible to stop print out the INFO of specific api path?

### DISCUSSION THREAD

**Discussion by jhaoheng** [fastapi/fastapi#7213] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorial in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Example

```Python
from fastapi import FastAPI

app = FastAPI()


@app.get("/")
def read_root():
    return {"Hello": "World"}
```

### Description

- I have a health check service, and trigger `/` path every 30s and get 200 response, perfect!!
- And there have a lot of fastapi log `INFO` like below:

```
INFO:     127.0.0.1:64082 - "GET / HTTP/1.1" 200 OK
```

- Does it possible to hidden this info from the specific API Path, don't show it on the stdout?

### Environment

* OS: [Linux / macOS]
* FastAPI Version : 0.58.1

* Python version: Python 3.6.8 :: Anaconda, Inc.

**Comment by ArcLightSlavik** [comment, marked as answer]
Yes, but it's done by uvicorn / gunicorn not FastAPI.

Uvicorn: https://github.com/encode/starlette/issues/864
Gunicorn; https://stackoverflow.com/questions/57416645/disable-logging-in-gunicorn-for-a-specific-request-url-endpoint

**Comment by jhaoheng** [comment]
@ArcLightSlavik 

Thanks. I will check it.

**Comment by tiangolo** [comment]
Thanks for the help here @ArcLightSlavik! 🍰 

Thanks for closing the issue @jhaoheng ☕ 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
